app/component/TicketInformation.js

```diff
@@ -78,6 +80,28 @@ export default function TicketInformation(
     }
 
     const ticketUrl = () => {
+      const transitLegs = legs.filter(leg => leg.transitLeg);
+      let browserType;
+      if (isMobile && isIOS) {
+        browserType = 'ios';
+      } else if (isMobile && !isIOS) {
+        browserType = 'android';
+      } else {
+        browserType = 'web';
+      }
+      if (
+        config.ticketingUrls &&
+        config.ticketingUrls[browserType] &&
+        transitLegs.length > 0
+      ) {
+        let url = config.ticketingUrls[browserType];
+        const startTime = new Moment(transitLegs[0].startTime);
+        url = url.replace('{startStopName}', transitLegs[0].from.name);
```

Comment:
Does it really depend on the _name_? That does seem quite brittle.

Reply:
I see what is happening here...

![Uploading Screenshot from 2022-03-18 09-37-32.png…]()

I guess there isn't a better option available, isn't it?